processor/executor/src/main/java/com/redhat/service/bridge/processor/actions/kafkatopic/KafkaTopicActionInvoker.java

```diff
@@ -1,26 +1,26 @@
-package com.redhat.service.bridge.actions.kafkatopic;
+package com.redhat.service.bridge.processor.actions.kafkatopic;
 
 import org.eclipse.microprofile.reactive.messaging.Emitter;
 import org.eclipse.microprofile.reactive.messaging.Message;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.redhat.service.bridge.actions.ActionInvoker;
 import com.redhat.service.bridge.infra.models.dto.ProcessorDTO;
+import com.redhat.service.bridge.processor.actions.ActionInvoker;
 
 import io.smallrye.reactive.messaging.kafka.api.OutgoingKafkaRecordMetadata;
 
-public class KafkaTopicInvoker implements ActionInvoker {
+public class KafkaTopicActionInvoker implements ActionInvoker {
 
-    private static final Logger LOG = LoggerFactory.getLogger(KafkaTopicInvoker.class);
+    private static final Logger LOG = LoggerFactory.getLogger(KafkaTopicActionInvoker.class);
 
     private final String topic;
 
     private final ProcessorDTO processor;
```

Comment:
That's a nice observation but I am not changing functionality with this refactoring so I'd keep it there since it was there before. We can think about this separately.